bundles/org.openhab.ui/web/src/components/cards/glance/location/status-badge.vue

```diff
@@ -70,6 +71,10 @@ export default {
     query () {
       let direct, equipment, allPoints, points
       switch (this.type) {
+        case 'battery':
+          direct = findPoints(this.element.properties, 'Point_LowBattery', true)
```

Comment:
Should we check for property "Energy" too?

```suggestion
          direct = findPoints(this.element.properties, 'Point_LowBattery', true, 'Property_Energy')
```

Reply:
> Should we check for property "Energy" too?

I think that's superfluous since the point class alone is self-describing.

Also it's `Point_Status_LowBattery`, not `Point_LowBattery`.